src/model/channel/mod.rs

```diff
@@ -400,26 +333,22 @@ pub enum ChannelType {
     ///
     /// [`GuildChannel`]: struct.GuildChannel.html
     Voice = 2,
-    /// An indicator that the channel is the channel of a [`Group`].
-    ///
-    /// [`Group`]: struct.Group.html
-    Group = 3,
     /// An indicator that the channel is the channel of a [`ChannelCategory`].
     ///
     /// [`ChannelCategory`]: struct.ChannelCategory.html
-    Category = 4,
+    Category = 3,
```

Comment:
These numbers shouldn't be changed. [Category channels don't have a `type` of `3`, groups do.](https://discordapp.com/developers/docs/resources/channel#channel-object-channel-types)